isis/src/base/objs/PvlTranslationTable/PvlTranslationTable.cpp

```diff
@@ -340,22 +352,17 @@ namespace Isis {
    * Returns the input keyword name from the translation table corresponding to
    * the output name argument.
    *
-   * @param nName The output name to be used to search the translation table.
+   * @param translationGroupName The output name to be used to search the translation table.
    *
    * @return QString The input keyword name
    *
-   * @throws iException::Programmer
+   * @throws IException::Programmer
```

Comment:
Do we document the exception here with `@throws`, if the exception is actually thrown from `findTranslationGroup`?

Reply:
Nope. We shouldn't document it if this method doesn't throw it. Good catch.